api-gateway-config/conf.d/managed_endpoints.conf

```diff
@@ -67,21 +68,25 @@ server {
         set $cors_methods '';
         
         access_by_lua_block {
-            local routing = require "routing"
-            routing.processCall()
-            local cors = require "cors"
-            ngx.var.cors, ngx.var.cors_methods = cors.processCall(ngx.var["tenant"], ngx.var["gatewayPath"])   
+            if ngx.var.routing == nil then
```

Comment:
when lua_code_cache is turned `on` lua modules are not loaded from scratch; there is 1 lua vm per nginx worker process. If there's a difference in this I'm thinking that there might be other factors that can influence the performance of the import.
See more at https://github.com/openresty/lua-nginx-module#lua_code_cache